docs/src/packages/solvers.md

```diff
@@ -4,10 +4,22 @@ This section of the documentation contains brief documentation for some of the
 solvers that JuMP supports. The list of solvers is not exhaustive, but instead
 is intended to help you discover commonly used solvers.
 
+## Affiliation
+
 Packages beginning with `jump-dev/` are developed and maintained by the
-JuMP developers. Other packages are developed independently.
+JuMP developers. In many cases, these packages wrap external solvers that are
+not developed by the JuMP developers and, while the Julia packages are all
+open-source, in some cases the solvers themselves are closed source commercial
+products.
+
+Packages that do not begin with `jump-dev/` are developed independently, and the
+JuMP developers have no affiliation with their developers or control over their
```

Comment:
> or control over their contents

That's a little strange to say when we submitted PRs to set up the READMEs. We also pick which versions to include and nominally sanity check the contents before including it in the JuMP repo.

Maybe another way phrase this is that the developers of these packages requested or consented to the inclusion of the README contents in the JuMP documentation for the benefit of users.